aea/helpers/fingerprint.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
```

Comment:
I moved some methods to helpers module since the `ipfs_hash.py` file was getting populated with too many helper methods.